### ISSUE TITLE
Separate title and subtitle updates

### ISSUE THREAD

**Issue by IamMusavaRibica** [PrismarineJS/mineflayer#2715]
## Is your feature request related to a problem? Please describe.
My feature request is not related to a problem.

## Describe the solution you'd like
There should be separate events for when title is changed and when the subtitle is changed. Currently, [the event is fired when any of them change](https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/title.js#L5), but whether it was the title or the subtitle, is not specified. The 'title' event currently only knows about the new text, but changing that event would not make it backwards compatible. I open this issue for discussion on that

## Describe alternatives you've considered
None

**Comment by rob9315** [comment]
aside from that this event won't even be fired anymore for 1.19 and possibly below versions because title and subtitle has been separated into [two](https://wiki.vg/Protocol#Set_Title_Text) [packets](https://wiki.vg/Protocol#Set_Subtitle_Text)

**Comment by rob9315** [comment]
backwards compatibility for this is probably not a big deal as i doubt anyone depended on this wrong behavior considering it's already been broken for quite a while, i propose the event should be split into `title` and `subtitle`, where each carries the expected data.

**Comment by IamMusavaRibica** [comment]
#610 was when the current changes were made
For a bot I'm building, from the provided data, I can easily tell what is a title and what is a subtitle. But it is not different in the code, which is not practical

For the two events, in earlier version it was the same packet so those cases should be reviewed too

**Comment by rob9315** [comment]
yes, the old packet should trigger 'title' and 'subtitle' when appropriate

**Comment by SMEDjs** [comment]
https://github.com/PrismarineJS/mineflayer/pull/3498
